content/docs/use-cases/dvc.md

```diff
@@ -22,6 +22,9 @@ $ source .venv/bin/activate
 $ pip install -r requirements.txt
 ```
 
+This will install DVC with `pip`. Learn about other ways to install DVC
```

Comment:
Why not mention DVC on the line 17? (MLEM and DVC) ...

`This command also installs DVC ... it's not required to have DVC as a dependency, this is done for simplicity ... there are other options to install DVC` - something along these lines could be a bit better, wdyt?

Reply:
1. Moved that above as you suggested.
2. `it's not required to have DVC as a dependency, this is done for simplicity` -- not true, since to use MLEM models you'll need to have DVC installed: model/data binaries are uploaded to cloud here.

Reply:
Ah, you right indeed! Checked the implementation, and MLEM uses DVC API. Let's add `mlem[dvc, dvc-s3]` then.

Reply:
https://github.com/iterative/mlem/pull/343

Reply:
Ok, now you could do `pip install mlem[dvc, dvc-s3]`. Should we mention now the other ways to install DVC? (I also need to update MLEM generation code in `example-repos-dev` to have this in `requirements.txt`). @shcheklein

Reply:
Ok, @jorgeorpinel suggested below to remove this mention. I agree, so I removed it. I think we can merge this now. Please approve if you agree.

Reply:
Sorry, let me update this PR first. Posted a comment below.